src/tests/functional/shared_test_classes/src/single_op/binary_convolution.cpp

```diff
@@ -0,0 +1,90 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_op/binary_convolution.hpp"
+
+#include "ngraph_functions/builders.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+
+namespace ov {
+namespace test {
+std::string BinaryConvolutionLayerTest::getTestCaseName(const testing::TestParamInfo<binaryConvolutionTestParamsSet>& obj) {
+    binConvSpecificParams bin_conv_params;
+    ov::element::Type model_type;
+    ov::element::Type in_type, out_type;
+    std::vector<InputShape> shapes;
+    std::string target_device;
+
+    std::tie(bin_conv_params, model_type, in_type, out_type, shapes, target_device) = obj.param;
+
+    ov::op::PadType pad_type;
+    std::vector<size_t> kernel, stride, dilation;
+    std::vector<ptrdiff_t> pad_begin, padEnd;
+    size_t conv_out_channels;
+    float pad_value;
+    std::tie(kernel, stride, pad_begin, padEnd, dilation, conv_out_channels, pad_type, pad_value) = bin_conv_params;
+
+    std::ostringstream result;
+    result << "IS=(";
+    for (const auto& shape : shapes) {
+        result << ov::test::utils::partialShape2str({shape.first}) << "_";
+    }
+    result << ")_TS=(";
+    for (const auto& shape : shapes) {
+        for (const auto& item : shape.second) {
+            result << ov::test::utils::vec2str(item) << "_";
```

Comment:
Hmm, in general we can avoid using this method and use: 
```suggestion
            result << ov::Any(item).as<std::string>() << "_";
```